src/stratis_cli/_actions/_debug.py

```diff
@@ -91,6 +93,49 @@ def get_object_path(namespace):
         )
         print(pool_object_path)
 
+    @staticmethod
+    def get_metadata(namespace):
+        """
+        Get some information about the pool-level metadata.
+        """
+        # pylint: disable=import-outside-toplevel
+        from ._data import ObjectManager, Pool, pools
+
+        proxy = get_object(TOP_OBJECT)
+        managed_objects = ObjectManager.Methods.GetManagedObjects(proxy, {})
+
+        props = (
+            {"Uuid": namespace.uuid.hex}
+            if namespace.name is None
+            else {"Name": namespace.name}
+        )
+        (pool_object_path, _) = next(
+            pools(props=props).require_unique_match(True).search(managed_objects)
+        )
+
+        (metadata, return_code, message) = Pool.Methods.Metadata(
+            get_object(pool_object_path), {"current": not namespace.written}
+        )
+
+        if return_code != StratisdErrors.OK:  # pragma: no cover
+            raise StratisCliEngineError(return_code, message)
+
+        metadata = [int(x) for x in metadata]
+
+        if namespace.format is MetadataFormat.RAW:
+            print(metadata)
+            return
```

Comment:
What is the case you were thinking of when adding the raw option? Is it for if the JSON doesn't parse? I'm not entirely against it, but I think that it might be good to consider whether this will be used frequently.

Reply:
@jbaublitz Since we're returning a string from stratisd, we can just drop the raw and have only two options, pretty or not.